tests/fault_injector/test_platform_blocked_fault.py

```diff
@@ -74,9 +74,9 @@ def test_resolve_platform_blocked_fault(
         # pylint: disable-next=unused-argument
         combine_platform_and_wrapper,
     ):
-        with pytest.raises(NotImplementedError):
-            platform_blocked_fault.inject_fault(tick)
+        platform_blocked_fault.inject_fault(tick)
         assert platform.blocked
-        with pytest.raises(NotImplementedError):
-            platform_blocked_fault.resolve_fault(tick)
+        assert platform_blocked_fault.interlocking.route_controller.method_calls > 0
+        platform_blocked_fault.resolve_fault(tick)
+        assert platform_blocked_fault.interlocking.route_controller.method_calls > 1
```

Comment:
Again, why not use exact numbers?
```suggestion
        assert platform_blocked_fault.interlocking.route_controller.method_calls == 1
        platform_blocked_fault.resolve_fault(tick)
        assert platform_blocked_fault.interlocking.route_controller.method_calls == 2
```